examples/configure_hydra/job_override_dirname/config.yaml

```diff
@@ -8,6 +8,6 @@ hydra:
         exclude_keys:
           - seed
 
-a: a1
-b: b1
+lr: 0.1
+optimizer: adam
```

Comment:
optimizer is typically a group.
How about:
```yaml
learning_rate: 0.1
batch_size: 32
```

Reply:
sure, updated.